gulp-config.js

```diff
@@ -1,17 +1,19 @@
-module.exports = function () {
-  var instanceRoot = "C:\\websites\\Habitat.dev.local";
-  var config = {
-    websiteRoot: instanceRoot + "\\Website",
-    sitecoreLibraries: instanceRoot + "\\Website\\bin",
-    licensePath: instanceRoot + "\\Data\\license.xml",
-    solutionName: "Habitat",
-    buildConfiguration: "Debug",
-    buildToolsVersion: 15.0,
-    buildMaxCpuCount: 0,
```

Comment:
Should default buildMaxCpuCount to 1 and let others change it to 0 (auto) if they choose to. Many issues have been reported where "file in use" or locked files came up when trying to do parallel builds.